spec/services/user_signup_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'spec_helper'
+
+describe UserSignup do
+  describe "#sign_up" do
+    context "valid personal info and valid card" do
+      let(:charge) { double(:charge, successful?: true) }
+      before { StripeWrapper::Charge.should_receive(:create).and_return(charge) }
+      after { ActionMailer::Base.deliveries.clear }
+
+      it "creates the user" do
+        UserSignup.new(Fabricate.build(:user)).sign_up(stripe_token: "some_stripe_token")
+        expect(User.count).to eq(1)
+      end
+
+      it "makes the user follow the inviter" do
+        user1 = Fabricate(:user)
+        invitation = Fabricate(:invitation, inviter: user1)
+        UserSignup.new(Fabricate.build(:user, email: invitation.recipient_email))
+          .sign_up(stripe_token: "some_stripe_token", invitation_token: invitation.token)
+        user2 = User.find_by(email: invitation.recipient_email)
+        expect(user2.follows?(user1)).to be_true
+      end
+
+      it "makes the inviter follow the user" do
+        user1 = Fabricate(:user)
+        invitation = Fabricate(:invitation, inviter: user1)
+        UserSignup.new(Fabricate.build(:user, email: invitation.recipient_email))
+          .sign_up(stripe_token: "some_stripe_token", invitation_token: invitation.token)
```

Comment:
Line is too long. [89/80]<br>Place the . on the previous line, together with the method call receiver.